cmd/eksctl-anywhere/cmd/installpackages.go

```diff
@@ -0,0 +1,104 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/config"
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+type installPackageOptions struct {
+	source      curatedpackages.BundleSource
+	kubeVersion string
+	name        string
+	registry    string
+}
+
+var ipo = &installPackageOptions{}
+
+func init() {
+	installCmd.AddCommand(installPackageCommand)
+	installPackageCommand.Flags().Var(&ipo.source, "source", "Location to find curated packages: (cluster, registry)")
+	if err := installPackageCommand.MarkFlagRequired("source"); err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+	installPackageCommand.Flags().StringVar(&ipo.kubeVersion, "kubeversion", "", "Kubernetes Version of the cluster to be used. Format <major>.<minor>")
+	installPackageCommand.Flags().StringVar(&ipo.name, "name", "", "Custom name of the curated package to install")
+	if err := installPackageCommand.MarkFlagRequired("name"); err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+}
+
+var installPackageCommand = &cobra.Command{
+	Use:          "packages [flags]",
+	Aliases:      []string{"package", "packages"},
+	Short:        "Install package(s)",
+	Long:         "This command is used to Install a curated package. Use list to discover curated packages",
+	PreRunE:      preRunPackages,
+	SilenceUsage: true,
+	RunE:         runInstallPackages(),
+}
+
+func runInstallPackages() func(cmd *cobra.Command, args []string) error {
```

Comment:
Do we need to return a closure or can we just
```suggestion
func runInstallPackages(cmd *cobra.Command, args []string) error {
```

Reply:
I didn't push the change but this has been fixed before. Sorry for the confusion.